monkestation/code/game/objects/items/anime.dm

```diff
@@ -0,0 +1,70 @@
+//FOR THE BASE OBJECT//
+
+/obj/item/anime
+	name = "anime dermal implant"
+	desc = "You should not be seeing this item!"
+	icon = 'monkestation/icons/obj/device.dmi'
+	icon_state = "coder"
+	var/obj/item/organ/ears/ears = null
+	var/obj/item/organ/tail/tail = null
+
+/obj/item/anime/examine(mob/user)
+	. = ..()
+	. += "Ctrl+Click to adjust the color."
+
+/obj/item/anime/attack_self(mob/living/carbon/human/user)
+	var/old_hair_color = user.hair_color
+	user.hair_color = sanitize_hexcolor(src.color) //I guess I have to do this fuck living code
+	if(ears)
+		ears.Insert(user)
+	if(tail)
+		tail.Insert(user)
+	user.hair_color = old_hair_color
+	var/turf/location = get_turf(user)
+	user.add_splatter_floor(location)
+	var/msg = "<span class=danger>You feel the power of God and Anime flow through you! </span>"
+	to_chat(user, msg)
```

Comment:
Some kind of sound, probably taken from the autosurgeon, would make sense here.

Reply:
added